src/Tribe/Linked_Posts.php

```diff
@@ -1177,7 +1177,7 @@ class="tribe-dropdown linked-post-dropdown hide-before-select2-init"
 				<?php if ( $creation_enabled ) : ?>
 				data-freeform
 				data-sticky-search
-				data-create-choice-template="<?php echo __( 'Create: <b><%= term %></b>', 'the-events-calendar' ); ?>"
+				data-create-choice-template="<?php echo __( 'Create: <%= term %>', 'the-events-calendar' ); ?>"
```

Comment:
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '__'.
* :warning: Found Underscore.js unescaped output notation: "<%=".
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 3ed4a2afc8a7baf82a167423d500b65f</sub>